RxJavaTemplate/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:${Versions.BUILD_GRADLE_VERSION}")
+        classpath("com.google.dagger:hilt-android-gradle-plugin:${Versions.HILT_VERSION}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN_VERSION}")
+        classpath("androidx.navigation:navigation-safe-args-gradle-plugin:${Versions.NAVIGATION_VERSION}")
+    }
+}
+
+plugins {
+    id("io.gitlab.arturbosch.detekt").version(Versions.DETEKT_VERSION)
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        maven { url = uri("https://jitpack.io") }
+        mavenCentral()
+    }
+}
+
+tasks.register("clean", Delete::class) {
+    delete(rootProject.buildDir)
+}
+
+detekt {
+    toolVersion = Versions.DETEKT_VERSION
+
+    input = files("app/src/main/java", "data/src/main/java")
+    parallel = false
+    config = files("detekt-config.yml")
+    buildUponDefaultConfig = false
+    disableDefaultRuleSets = false
+
+    debug = false
+    ignoreFailures = false
+
+    ignoredBuildTypes = listOf("release")
+    ignoredFlavors = listOf("production")
+
+    reports {
+        xml {
+            enabled = true
+            destination = file("build/reports/detekt.xml")
+        }
+        html {
+            enabled = true
+            destination = file("build/reports/detekt.html")
+        }
+    }
+}
```

Comment:
Ah I just see, EOF issue again. 🙏🏻 
Or you can let AS manages it for you. 😉 
![Screen Shot 2564-08-31 at 11 29 20](https://user-images.githubusercontent.com/12026942/131441998-7f8e8b09-c140-4bfe-9923-59437410f57d.png)

Reply:
@lydiasama I disabled the settings on AS because of the new project doesn’t use that style 😄  Updated btw 🚀

Reply:
@hoangnguyen92dn I see.